src/main/kotlin/org/ozwillo/dcimporter/model/sirene/Organization.kt

```diff
@@ -6,6 +6,7 @@ import org.ozwillo.dcimporter.util.DCUtils
 import org.ozwillo.dcimporter.util.soap.response.parsing.ResponseObject
 
 data class Organization(
+    val uri: String = "",
```

Comment:
I added the uri for not having to do it in front, and it seems normal to me to return the id of the organization

Reply:
The API don't exists yet. But keep the idea for later. Get organization by its uri is safer.